### PR TITLE
Allow vmi pause state change timeout to include initial CRD state

### PR DESCRIPTION
**Pull request by sarahbx**
##### Short description:
This change allows the state change from paused-to-unpaused or vice-versa be with in the timeout period.

### CODE REVIEW

File: ocp_resources/virtual_machine.py [hunk 1]
```diff
@@ -292,6 +292,11 @@ def wait_for_vmi_condition_pause_status(self, pause, timeout=TIMEOUT):
             func=self.get_vmi_active_condition,
         )
         for sample in samples:
+            # VM in state change
+            if (pause and not sample.get("reason")) or (
+                sample.get("reason") == "PausedByUser" and not pause
+            ):
+                continue
```
Comment (myakove) [inline ocp_resources/virtual_machine.py:299]: Sorry, but I don't understand what should be the results here
Can you please add comments for each and explain when and why this is needed?

maybe somthing like:
xxx = pause and not sample.get("reason")  # explain ......
yyy = sample.get("reason") == "PausedByUser"  # explain ......

and please elaborate more about "# VM in state change"

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/virtual_machine.py (hunk 1 above)
Comment (sarahbx) [reply ocp_resources/virtual_machine.py:303]: Comment updated

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**